webapp/components/CategoriesSelect/CategoriesSelect.vue

```diff
@@ -65,14 +65,34 @@ export default {
     categoryButtonsId(categoryId) {
       return `category-buttons-${categoryId}`
     },
+    sortCategories(categories) {
+      const misc = categories.find((cat) => cat.slug === 'miscellaneous')
+      const sortedCategories = categories
+        .filter((cat) => cat.slug !== 'miscellaneous')
```

Comment:
reference `misc` here?

```suggestion
        .filter((cat) => cat.slug !== 'miscellaneous')
```
```suggestion
        .filter((cat) => cat.slug !== misc.slug)
```